charon-models/src/test/java/gov/va/api/lighthouse/charon/models/lhslighthouserpcgateway/LhsLighthouseRpcGatewayResponseTest.java

```
@@ -29,9 +30,20 @@ void entryValueHelpers() {
     assertThat(e.internal("here", Integer::valueOf).get()).isEqualTo(2);
     assertThat(e.external("here", Integer::valueOf).get()).isEqualTo(1);
 
+    e.fields().put("where", Values.of("x", "i"));
+
+    assertThat(e.internal("where", Map.of("i", "INTERNAL")).get()).isEqualTo("INTERNAL");
+    assertThat(e.external("where", Map.of("x", "EXTERNAL")).get()).isEqualTo("EXTERNAL");
```

^ these two asserts are no different.
In general, expand your data set to include more than one possible value. For example, I could pass your test with dump implementation
```
map.values().stream().findFirst().get()
```

My point is that test data sets of 1 can hide errors. Something as simple as this can suffice:

```
var enumeration = Map.of("i", "INTERNAL", "x", "EXTERNAL")
assertThat(e.internal("where", enumeration).get()).isEqualTo("INTERNAL");
... use enumeration some more ...
```